### PR TITLE
Special Christmas edition: RC testing 🎄

### PR DESCRIPTION
**Pull request by jnewbery**

### CODE REVIEW

**Comment by jnewbery** [comment]
@jarolrod @adamjonas

File: _posts/2020-12-23-rc-testing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  (rc1) is tagged. The rc is then tested. If any issues are found, fixes are
+  merged into the branch and a new rc is tagged. This continues until no major
+  issues are found in an rc, and that rc is then considered to be the final
+  release version.
+
+- To ensure that users don't experience issues with the new software, it's
+  essential that the rcs are thoroughly tested. This special review club
+  meeting is for people who want to help with that review process.
+
+- The [Bitcoin Core Release Candidate Testing
```
Comment (jonatack) [inline _posts/2020-12-23-rc-testing.md:25]: s/The/This/

File: _posts/2020-12-23-rc-testing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Guide](https://gist.github.com/jarolrod/6495932dc6b598ddef49e6374b348f4e) has
+  full instructions for testing the release candidate.
```
Comment (jonatack) [inline _posts/2020-12-23-rc-testing.md:27]: s/full/a tutorial of|some/

File: meetings-authors.html [hunk 3]
```diff
@@ -45,7 +45,7 @@ <h3 class="meetings-index-section">
                 &raquo;
               </td>
               <td>
-                <a class="Home-posts-post-title" href="{{ post.url }}">#{{ post.pr }} {{ post.title }}</a>
+		      <a class="Home-posts-post-title" href="{{ post.url }}">{% if post.pr %}#{{ post.pr }} {% endif %}{{ post.title }}</a>
```
Comment (jonatack) [inline meetings-authors.html:48]: indentation is off due to use of tabs in the change, instead of spaces

File: meetings-components.html [hunk 4]
```diff
@@ -38,7 +38,7 @@ <h3 class="meetings-index-section">{{component}}</h3>
                   &raquo;
                 </td>
                 <td>
-                  <a class="Home-posts-post-title" href="{{ post.url }}">#{{ post.pr }} {{ post.title }}</a>
+		  <a class="Home-posts-post-title" href="{{ post.url }}">{% if post.pr %}#{{ post.pr }} {% endif %}{{ post.title }}</a>
```
Comment (jonatack) [inline meetings-components.html:41]: idem

File: meetings-hosts.html [hunk 5]
```diff
@@ -43,7 +43,7 @@ <h3 class="meetings-index-section">
                 &raquo;
               </td>
               <td>
-                <a class="Home-posts-post-title" href="{{ post.url }}">#{{ post.pr }} {{ post.title }}</a>
+		<a class="Home-posts-post-title" href="{{ post.url }}">{% if post.pr %}#{{ post.pr }} {% endif %}{{ post.title }}</a>
```
Comment (jonatack) [inline meetings-hosts.html:46]: idem

**Review by jonatack — COMMENTED** [review]
Yay, the review club is back!

**Comment by jonatack** [comment]
The PR link in the `/rc-testing` page is bad, as no PR number is provided: https://github.com/bitcoin/bitcoin/pull/

File: _posts/2020-12-23-rc-testing.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- When all of the PRs for a release have been merged, _Release Candidate 1_
+  (rc1) is tagged. The rc is then tested. If any issues are found, fixes are
+  merged into the branch and a new rc is tagged. This continues until no major
+  issues are found in an rc, and that rc is then considered to be the final
+  release version.
+
+- To ensure that users don't experience issues with the new software, it's
+  essential that the rcs are thoroughly tested. This special review club
+  meeting is for people who want to help with that review process.
```
Comment (jonatack) [inline _posts/2020-12-23-rc-testing.md:23]: maybe s/review/vital review/

**Comment by jonatack** [comment]
In the testing guide itsely, in the tor v3 section:

- testers should launch bitcoind with the `-debug=tor` configuration option to see useful info on the onion configuration

- `./src/bitcoin-cli -datadir=/tmp/21-rc-test getpeerinfo` could be replaced or complemented by `./src/bitcoin-cli -datadir=/tmp/21-rc-test -netinfo 4`, which is getpeerinfo + getnetworkinfo for humans and provides more useful feedback

**Comment by jonatack** [comment]
In the testing guide itself, in the tor v3 section:

- testers should launch bitcoind with the `-debug=tor` configuration option to see useful info on the onion configuration

- `./src/bitcoin-cli -datadir=/tmp/21-rc-test getpeerinfo` could be replaced or complemented by `./src/bitcoin-cli -datadir=/tmp/21-rc-test -netinfo 4`, which is getpeerinfo + getnetworkinfo for humans and provides more useful feedback

File: _posts/2020-12-23-rc-testing.md (hunk 1 above)
Comment (jnewbery) [reply _posts/2020-12-23-rc-testing.md:25]: fixed.

File: _posts/2020-12-23-rc-testing.md (hunk 2 above)
Comment (jnewbery) [reply _posts/2020-12-23-rc-testing.md:27]: fixed

File: meetings-authors.html (hunk 3 above)
Comment (jnewbery) [reply meetings-authors.html:48]: fixed

File: meetings-components.html (hunk 4 above)
Comment (jnewbery) [reply meetings-components.html:41]: fixed

File: meetings-hosts.html (hunk 5 above)
Comment (jnewbery) [reply meetings-hosts.html:46]: fixed

File: _posts/2020-12-23-rc-testing.md (hunk 6 above)
Comment (jnewbery) [reply _posts/2020-12-23-rc-testing.md:23]: good!

**Comment by jnewbery** [comment]
> The PR link in the /rc-testing page is bad, as no PR number is provided: bitcoin/bitcoin/pull

Fixed

**Comment by jnewbery** [comment]
Thanks for the quick review @jonatack!

**Comment by jonatack** [comment]
@jarolrod I commented in your gist, feel free to add it directly to the instructions: 

https://gist.github.com/jarolrod/6495932dc6b598ddef49e6374b348f4e#gistcomment-3568999

**Comment by jonatack** [comment]
Tested and content ACK

File: _posts/2020-12-23-rc-testing.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+---
+layout: pr
+date: 2020-12-16
```
Comment (adamjonas) [inline _posts/2020-12-23-rc-testing.md:3]: 2020-12-23

**Comment by adamjonas** [comment]
ACK after the date change

File: _posts/2020-12-23-rc-testing.md (hunk 7 above)
Comment (jonatack) [reply _posts/2020-12-23-rc-testing.md:3]: good catch

**Comment by jarolrod** [comment]
ACK, super excited 🚀

**Comment by jonatack** [comment]
Is this really happening in 2 days? Seems a bit short warning?

File: _posts/2020-12-23-rc-testing.md (hunk 7 above)
Comment (jnewbery) [reply _posts/2020-12-23-rc-testing.md]: fixed

**Comment by jnewbery** [comment]
Merged & tweeted: https://twitter.com/BitcoinCorePRs/status/1341159717146677249

**Comment by adamjonas** [comment]
> Is this really happening in 2 days?

It's a 🎄 miracle!

**Pull request merged**